skrifa/src/color/transform.rs

```diff
@@ -25,6 +27,23 @@ pub struct Transform {
     pub dy: f32,
 }
 
+impl MulAssign for Transform {
```

Comment:
I realize we don’t strictly need it here but this will be exposed publicly and having an implementation of MulAssign without Mul feels a bit weird.

Reply:
Added a `Mul` implementation, moved the logic into that and made the `MulAssign` implementation based on the former - hope that's what's usually done. Not sure I need the `clone()` there, but I guess I need some sort of temporary value at least?

Reply:
We could also derive `Copy` for `Transform` which would remove the need for the clone. I’m good either way.

Reply:
I'll add that.